SR2023/2022-10-05-place-confirmation.md

```diff
@@ -0,0 +1,23 @@
+---
+to: Student Robotics 2023
+subject: Confirming your place at Student Robotics 2023
+note: This email will be sent multiple times
+---
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2023!
+
+At kickstart we'll run through the competition, the kit, and of course this year's game. Kickstart will be held in Southampton and London on Saturday 22nd October.
+
+{% if requested southampton %}
+We're expecting you at the [Southampton](https://studentrobotics.org/events/sr2023/southampton-kickstart/) kickstart.
+{% if requested london %}
+We're expecting you at the [London](https://studentrobotics.org/events/sr2023/london-kickstart/) kickstart.
+{% if southampton or london %}
+In the original sign-up, you requested a place at either the [Southampton](https://studentrobotics.org/events/sr2023/southampton-kickstart/) or [London](https://studentrobotics.org/events/sr2023/london-kickstart/). Please let us know which kickstart you'd like to attend so we can ensure there's a kit ready for you.
```

Comment:
I don't think we should hold this email back for the teams who've said Southampton or London, instead I think we should tell them what information we have and that we're aiming to confirm whether London will happen (and the venue etc.) by the end of the week.